README.rst

```diff
@@ -8,7 +8,24 @@ A Python client library for working with the DSpace API. This was written agains
 Usage
 -----
 
-This project has not been released for active use yet. Check back soon!
+To authenticate the DSpace client::
+
+  from dspace.bitstream import Bitstream
+  from dspace.client import DSpaceClient
+  from dspace.item import Item, MetadataEntry
+  
+  client = DSpaceClient(<DSpace API URL>)
+  client.login(<your email>, <your password>)
+
+To post an item and associated bitstream with an authenticated client::
+  
+  title = MetadataEntry(key="dc.title", value="Test Item")
+  item = Item(metadata=[title])
+  item.post(client, collection_handle=response["uuid"])
```

Comment:
This looks like a copy/paste error. `collection_handle=<a valid DSpace collection handle>` would probably make more sense.

Reply:
Ugh, totally correct, a leftover from how I originally structured the example